blog/2022-07-22-vs-iac.mdx

```diff
@@ -0,0 +1,48 @@
+
+[IaSQL](https://iasql.com), like IaC tools, allows creating, updating or deleting infrastructure in the cloud. IaSQL does this by using the actual representation of the infrastructure instead of a blueprint. This provides several benefits which we will cover, but primordially it allows developer to model cloud infrastructure in the same way the information in most software systems is represented: as data in a database. This means developers can scale the cloud with a familiar set of developer tools that the Postgres ecosystem has built over many decades, and with one of the most well-known programming languages, SQL.
+
+### Statefiles vs Databases
+
+IaC tools like Terraform and Pulumi automatically create statefiles to reconcile infrastructure that exists in the cloud, but not in the infrastructure declaration. Statefiles don't use human-readable syntax and are hard to inspect or modify, especially at scale. Terraform has gone through great lengths to make it palatable to manipulate these machine generated files using their CLI tool. However, databases were created to replace files when manipulating a large amount of structured information and developer communities have built a lot of tooling around them. IaSQL sets forth a much more natural and battle-tested way to manage infrastructure data.
+
+Some organizations prevent the generation of statefiles when using IaC tools by making cloud dashboards read-only for all employees and only allowing infrastructure changes to occur through the declaration. However, this means every single team in an organization that can span thousands of people and software systems has to adhere to this convention. This can be an untenable premise when workflows span multiple clouds, time zones, deployment strategies, dependencies and versions. Application engineers often need to move quickly and create new infrastructure through a cloud UI that could also depend on infrastructure previously created through an IaC tool. To make matters worse, subsequent changes to the IaC declaration can affect the infrastructure created through the cloud UI as it is only represented in the statefile.
+
+We created the IaSQL `sync` command to let teams create resources outside of IaSQL while still managing said resources. This flexibility makes it possible to have multiple cloud management tools across different teams without getting locked into a single tool. When using IaSQL's `sync`, there is no need to constantly recreate changes or deal with state files to stay up to date with infrastructure changes.
+
+![](../static/img/comparison.jpg)
+
+### Importing existing infrastructure (WIP)
+
+IaSQL automatically imports all existing cloud resources when the initial 2-way binding is setup. 
```

Comment:
Just a couple of bullet point that might help here
- import all resources for the modules you installed? Maybe it make sense to introduce the modules concept?
- IaC isolation vs IaSQL complete view?